docs: Update usage documents (#1073)
spidernet-io/spiderpool: docs/usage/ippool-multi.md

# Multiple IPPool

*Spiderpool can specify multiple alternative IP pools for one IP allocation.*

## Set up Spiderpool

If you have not deployed Spiderpool yet, follow the guide [installation](https://github.com/spidernet-io/spiderpool/blob/main/docs/usage/install.md) for instructions on how to deploy and simply configure Spiderpool.

## Get Started

First, create two IPPools each containing 2 IP addresses.

```bash
kubectl apply -f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/ippool-multi/test-ipv4-ippools.yaml
```

> Create a Pod and have it allocate an IP address from these IPPools.

Reviewer: ```suggestion
Create a Pod and allocate an IP address to it from these IPPools.
```

default-ipv4-ippool   4         172.18.41.0/24   1                    2                false
```

> Then, create a Deployment with 2 replicas and have its Pods allocate IP addresses from the two IPPools above.

Reviewer 2: A little weird for this kind of pattern. Need help from @Michelle951.

> Then, create a Deployment with 2 replicas and have its Pods allocate IP addresses from the two IPPools above.

Reviewer: ```suggestion
Then, create a Deployment with 2 replicas and allocate IP addresses to its Pods from the two IPPools above.
```

        command: ["/bin/sh", "-c", "trap : TERM INT; sleep infinity & wait"]
```

Spiderpool will successively try to allocate IP addresses **in the order of** the elements in the "IP pool array" until the first allocation succeeds or all fail. Of course, you can specify the [pool selection rules](TODO) (that defines alternative IP pools) in many ways, the Pod annotation `ipam.spidernet.io/ippool` is used here to select IP pools.

> Finally, when the IP addresses in IPPool `default-ipv4-ippool` use out, the IPPool `backup-ipv4-ippool` takes over.

Reviewer: ```suggestion
Finally, when addresses in IPPool `default-ipv4-ippool` are used up, the IPPool `backup-ipv4-ippool` takes over.
```
